ChallengesWithTestsMarkVII/ChallengesSet05.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Text;
+
+namespace ChallengesWithTestsMarkVII
+{
+    public class ChallengesSet05
+    {
+        public int GetNextNumberDivisibleByN(int startNumber, int n)
+        {
+            if (startNumber == 0)
+            {
+                return n;
+            }
+
+            List<int> firstlist = new List<int>();
```

Comment:
The list usage in this method looks to be unnecessary. Also, if start `startNumber` and `n` are very large, this list could also become very large. For example, `GetNextNumberDivisibleByN(1000000000, 1000000000)` would result in a list with a billion elements being stored in memory, when it could be solved without a list